nixos/modules/security/pam.nix

```diff
@@ -634,6 +634,15 @@ let
           description = "Contents of the PAM service file.";
         };
 
+        forwardPass = lib.mkOption {
+          type = lib.types.bool;
+          default = false;
+          description = ''
+            The entered password is put on the stack for other
+            PAM modules to use.
+          '';
+        };
```

Comment:
This option should be specific to Google Authenticator, under the `security.pam.services.<name>.googleAuthenticator` option. The reason is that forwarding the password is not a generally applicable option; it is specific to different PAM rules, and users may wish to mix-and-match the settings.

As a bonus, if you do that, you can make the description more specific to Google Authenticator so that it's easier for users to understand what it does and why they should enable this option.

Reply:
Fixed.